pkg/apis/operator/v1alpha1/types.go

```diff
@@ -569,6 +569,8 @@ type CredentialsRotation struct {
 	// ETCDEncryptionKey contains information about the ETCD encryption key credential rotation.
 	// +optional
 	ETCDEncryptionKey *gardencorev1beta1.ETCDEncryptionKeyRotation `json:"etcdEncryptionKey,omitempty"`
+	// Observability contains information about the observability credential rotation.
+	Observability *gardencorev1beta1.ObservabilityRotation `json:"observability,omitempty"`
```

Comment:
`Observability` should be `optional` right? `CredentialsRotation` is an `optional` field.